hail/python/hail/backend/backend.py

```diff
@@ -411,7 +412,8 @@ def get_reference(self, name):
         return resp.json()
 
     def load_references_from_dataset(self, path):
-        raise NotImplementedError
+        # FIXME
```

Comment:
Fixme?

Reply:
I'm inclined to leave this broken with the FIXME for now.  There are a bunch of tasks related to reference genomes to get them working properly in the service, and this will be one of them.